src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -124,7 +124,11 @@ public without sharing class UTIL_MasterSchedulableHelper {
     public static void setScheduledJobs() {
 
         Map<String, CronTrigger> scheduledJobs = new Map<String, CronTrigger>();
-        
+
+        // always delete and reschedule all jobs in this particular scenario
+        Error_Settings__c orgErrorSettings = UTIL_CustomSettingsFacade.getOrgErrorSettings();
+        Boolean rescheduleAllJobs = (UTIL_Namespace.isSandbox() && orgErrorSettings.Sandbox_Jobs_Rescheduled__c == false);
```

Comment:
Does it make sense to put all criteria into this new boolean so we don't need to look at 2 different locations for the condition?